src/core/analytics.js

```diff
@@ -464,7 +476,10 @@ class Analytics {
    */
   page(category, name, properties, options, callback) {
     leaveBreadcrumb(`Page event`);
-    if (!this.loaded) return;
+    if (!this.loaded) {
+      this.toBeProcessedArray.push(['page', ...arguments]);
```

Comment:
can you please remind me why we did this one?

Reply:
is this tonot loose events if the sdk has loaded but still has not proceesed the config?

Reply:
To buffer the events that are triggered before SDK is loaded.